docs/core/linux-prerequisites.md

```diff
@@ -94,277 +112,467 @@ To install .NET Core on RHEL 7:
+    ```bash
+    sudo sh -c 'echo "deb [arch=amd64] https://apt-mo.trafficmanager.net/repos/dotnet-release/ trusty main" > /etc/apt/sources.list.d/dotnetdev.list'
+    sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys B02C46DF417A0893
+    sudo apt-get update
+    ```
+
+    **Ubuntu 16.04 / Linux Mint 18**
 
-To install .NET Core on Debian 8 or 9 (64 bit):
-1. Get the prerequisites. 
     ```bash
-    sudo apt-get install curl libunwind8 gettext
+    sudo sh -c 'echo "deb [arch=amd64] https://apt-mo.trafficmanager.net/repos/dotnet-release/ xenial main" > /etc/apt/sources.list.d/dotnetdev.list'
+    sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys B02C46DF417A0893
+    sudo apt-get update
     ```
-2. Download the .NET Core SDK binaries (tarball).
 
-# [.NET Core 2.x](#tab/netcore2x)   
+    **Ubuntu 16.10**
 
-```bash
-     curl -sSL -o dotnet.tar.gz https://aka.ms/dotnet-sdk-2.0.0-linux-x64
-```
+    ```bash
+    sudo sh -c 'echo "deb [arch=amd64] https://apt-mo.trafficmanager.net/repos/dotnet-release/ yakkety main" > /etc/apt/sources.list.d/dotnetdev.list'
+    sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys B02C46DF417A0893
+    sudo apt-get update
+    ```
 
-# [.NET Core 1.x](#tab/netcore1x)   
+3. Install .NET Core 1.x on Ubuntu or Linux Mint:
 
-```bash
-     curl -sSL -o dotnet.tar.gz https://go.microsoft.com/fwlink/?linkid=848826
-```
----
-3. Extract the .NET Core SDK binaries.
     ```bash
-    sudo mkdir -p /opt/dotnet && sudo tar zxf dotnet.tar.gz -C /opt/dotnet
+    sudo apt-get install dotnet-dev-1.0.4
     ```
-4. Add dotnet to your PATH.
+
+4. Run the `dotnet --version` command to prove the installation succeeded.
+
     ```bash
-    sudo ln -s /opt/dotnet/dotnet /usr/local/bin
+    dotnet --version
     ```
-5. Run the `dotnet --help` command to prove the installation succeeded.
 
-     ```bash
-     dotnet --help
-     ```
+---
+
+ ## Install .NET Core for Debian 8 or Debian 9 (64 bit)
+
+To install .NET Core on Debian 8 or Debian 9 (64 bit):
 
-## Install .NET Core for Fedora 24, 25, or 26 (64 bit)
+1. Remove any **previous preview** versions of .NET Core from your system.
 
-> [!Warning]
-> Before you start, remove any previous versions of .NET Core from your system.
+2. Get the prerequisites.
+
+   ```bash
+   sudo apt-get install curl libunwind8 gettext
+   ```
+
+> [!Note]
+> A user-controlled directory is required for Linux system installs from tar.gz.
+
+# [.NET Core 2.x](#tab/netcore2x)
+
+3. Download the .NET Core SDK binaries (tarball).
 
-To Install .NET Core for Fedora 26,25(.NET Core 2.x) or 24 (.NET Core 1.x): 
-1. Get the prerequisites. 
     ```bash
-    sudo dnf install libunwind libicu
+    curl -sSL -o dotnet.tar.gz https://aka.ms/dotnet-sdk-2.0.0-linux-x64
     ```
-2. Download the .NET Core SDK binary  (tarball).
+1. Extract the .NET Core SDK binaries.
+
+    ```bash
+    mkdir -p ~/dotnet && tar zxf dotnet.tar.gz -C ~/dotnet
```

Comment:
I can't decide which is better for the home dir: ~/ or $HOME. What do you think?

Reply:
There are situations where the ~ doesn't get expanded to Home but I believe $HOME always does. I can remove adding to path if its outside of scope.